package.json

```diff
@@ -42,7 +42,7 @@
   },
   "devDependencies": {
     "bluebird": "^3.3.1",
-    "eslint": "^2.4.0",
+    "eslint": "^3.6.0",
```

Comment:
What is wrong with that line? I think an ignore is easier if it can't be fixed with a follow up later. Generally upgrading eslint gives a lot of changes, this one was sort of small but still a bit.

Reply:
I agree with @miniscruff, in this PR's scope it's better not to upgrade the linter version and just add an ignore rule if the line is legit/needed and cannot easily be changed to please the linter.

If we decide that we should upgrade the linter version, then we can do it with a separate ticket and PR.